components/generic/react-table/components/table.tsx

```diff
@@ -0,0 +1,276 @@
+import React, { useMemo, memo, useEffect } from 'react'
+import {
+  useTable,
+  useFlexLayout,
+  useResizeColumns,
+  useSortBy,
+  Column,
+  HeaderGroup,
+} from 'react-table'
+import { usePrevious } from 'react-use'
```

Comment:
why do we need to add this?

Reply:
to prevent data flashing. it's a useful library with different common useful hooks